### PR TITLE
Detail content

### PR DESCRIPTION
**Pull request by mishrasur7**
### Description

Regarding the title in line 111 it says Finding items in an array and in the content of it is showing how to find the index of item in the array. It makes more sense if the title is about Finding index of items in an array since the content is about finding index of items in an array. It gives more understanding to readers. 

### Motivation

This will help readers to understand better about finding index of items in an array

### Related issues and pull requests

Fixes #123

### CODE REVIEW

File: files/en-us/learn/javascript/first_steps/arrays/index.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Items in an array are numbered, starting from zero. This number is called the it
 
 4. Try making some more modifications to your array examples before moving on. Play around a bit, and see what works and what doesn't.
 
-## Finding items in an array
+## Finding index of items in an array
 
 This is fine if you know the index of an item, but what if you don't? You can find the index of a particular item using the {{jsxref("Array.prototype.indexOf()","indexOf()")}} method. This takes an item as an argument and returns the index, or `-1` if the item was not found in the array:
```
Comment (bsmth) [inline files/en-us/learn/javascript/first_steps/arrays/index.md:113]:
```suggestion
If you don't know the index of an item, you can use the {{jsxref("Array.prototype.indexOf()","indexOf()")}} method.
The `indexOf()` method takes an item as an argument and will either return the item's index or `-1` if the item is not in the array:
```

File: files/en-us/learn/javascript/first_steps/arrays/index.md [hunk 2]
```diff
@@ -108,9 +108,10 @@ Items in an array are numbered, starting from zero. This number is called the it
 
 4. Try making some more modifications to your array examples before moving on. Play around a bit, and see what works and what doesn't.
 
-## Finding items in an array
+## Finding index of items in an array
```
Comment (bsmth) [inline files/en-us/learn/javascript/first_steps/arrays/index.md:111]:
```suggestion
## Finding the index of items in an array
```

**Review by bsmth — APPROVED** [review]
Thanks @mishrasur7 for your contribution 👍🏻

**Comment by mishrasur7** [comment]
Thanks for reviewing and approving.

**Pull request merged**